README.md

```diff
@@ -1,111 +1,108 @@
 # nm-configurator
 
-A tool capable of identifying & storing the relevant NetworkManager settings
-for a given host out of a pool of predefined desired configurations.
+nm-configurator (or NMC) is a CLI which makes it easy to generate and apply NetworkManager configurations.
 
-Typically used with [Combustion](https://documentation.suse.com/sle-micro/5.4/single-html/SLE-Micro-deployment/#cha-images-combustion) 
-in order to bootstrap multiple nodes using the same provisioning artefact instead of depending on different custom images per machine.
-
-## What are the prerequisites?
+## How to install it?
 
-### Desired network configurations per host
+### Standard method
 
-`nm-configurator` depends on having the desired network state for all known nodes beforehand.
+Each release is published with NMC already built for `amd64` and `arm64` Linux systems:
 
-[NetworkManager](https://documentation.suse.com/sle-micro/5.4/html/SLE-Micro-all/cha-nm-configuration.html) 
-is using connection profiles defined as files stored under `/etc/NetworkManager/system-connections`.
-These config files (*.nmconnection) can be generated using [nmstate](https://nmstate.io/features/gen_conf.html).
+For AMD64 / x86_64 based systems:
 
-Each file contains the desired state for a single network interface (e.g. `eth0`).
-Configurations for all interfaces for all known hosts must be generated using `nmstate`.
+```shell
+$ curl -o nmc -L https://github.com/suse-edge/nm-configurator/releases/latest/download/nmc-x86_64 
```

Comment:
The `latest` flag here is not currently functioning. Also, do you think we should create an aarch64-apple-darwin release for local development and testing? Reason being, the aarch64 binary that we ship in the releases is seemingly only functioning on Linux?
```
rdo@tiw nm-configurator % uname -a
Darwin tiw.local 22.5.0 Darwin Kernel Version 22.5.0: Thu Jun  8 22:22:23 PDT 2023; root:xnu-8796.121.3~7/RELEASE_ARM64_T6020 arm64
rdo@tiw nm-configurator % file nmc
nmc: ELF 64-bit LSB executable, ARM aarch64, version 1 (SYSV), statically linked, Go BuildID=7BwBJnNJlJRnUbl2thQQ/28-rOLaTmUNHHZPy7ci2/Xvgxmzw6_9BSBXz0oYxI/u3jZiVKC7Vy47aHuefd5, with debug_info, not stripped
rdo@tiw nm-configurator % ./nmc
zsh: exec format error: ./nmc
```
Whereas it works just fine on Linux:
```
rdo@tw-aarch64-build:~> uname -a
Linux tw-aarch64-build.rdo.wales 6.5.6-1-default #1 SMP PREEMPT_DYNAMIC Fri Oct  6 11:20:48 UTC 2023 (c97c2df) aarch64 aarch64 aarch64 GNU/Linux
rdo@tw-aarch64-build:~> ./nmc
INFO[2023-11-11T11:59:48Z] starting network manager configurator...
FATA[2023-11-11T11:59:48Z] failed to load static host configuration: open config/host_config.yaml: no such file or directory
```

Reply:
The `latest` flag is not working because we haven't tagged (and produced a release) yet. It'll work once we do so.

The binaries indeed are compiled for Linux x86_64 and aarch64 targets, not Darwin. Please take a look at [this](https://github.com/suse-edge/nm-configurator/pull/9#discussion_r1390672763) comment.

Reply:
I'd say to call it nmc-linux-x86_64 just in case (and do the same for aarch64)

Reply:
And maybe replace the instructions as per

```
curl -o nmc -L https://github.com/suse-edge/nm-configurator/releases/latest/download/nmc-linux-$(uname -m)
chmod +x nmc
```

Reply:
Nice suggestion, I'll go with this, thanks!